package.json

```diff
@@ -12,7 +12,7 @@
     "release:patch": "npm version patch && npm publish && git push --follow-tags",
     "serve": "node server.js --env=dev",
     "serve:dist": "node server.js --env=dist",
```

Comment:
Can we adjust the names of these commands slightly to be more obvious which one will use the AWS api and which one will use a local one?